.github/workflows/pull_request.yml

```diff
@@ -1,19 +1,31 @@
 name: Lint on pull request
 on:
+  workflow_dispatch:
   pull_request:
     branches:
       - master
 
 jobs:
   full_ci:
     runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9"]
     steps:
       # Get the code into the container
       - name: Checkout
         uses: actions/checkout@v2
+      # Setup Python
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      # Run install script
+      - name: Install dependencies
+        run: "scripts/install"
       # Lint the code
-      - name: Python linters Black Check
-        uses: jose-lpa/python-black-check-action@1.0.0
+      - name: Lint code with black
+        run: python3 -m black .
```

Comment:
We can update this line to: 

```bash 
$ make format
```

https://github.com/deta/deta-python/blob/54d329e9a4a6dd8eee0be3534197a025a48c2353/Makefile#L19

Reply:
You're right! Thanks